### PR TITLE
test: add test for zklendLoanEntity

### PR DESCRIPTION
**Pull request by Kaminar-i**
hello @djeck1432 kindly review :)

### CODE REVIEW

**Comment by djeck1432** [comment]
@Kaminar-i Please, fix pylint issue

File: apps/data_handler/tests/loan_state/test_zklend_loan_entity.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        zklend_loan_entity.collateral[token] = deposit_amount
+        
+        assert zklend_loan_entity.collateral[token] == zklend_loan_entity.deposit[token]
+
+    def test_token_settings(self):
+        """Test token settings configuration."""
+        assert hasattr(ZkLendLoanEntity, 'TOKEN_SETTINGS')
+        assert isinstance(ZkLendLoanEntity.TOKEN_SETTINGS, dict)
+
+if __name__ == '__main__':
```
Comment (djeck1432) [inline apps/data_handler/tests/loan_state/test_zklend_loan_entity.py:177]: Remove this part of code, it's not necessary to have it

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by Kaminar-i** [comment]
hello @djeck1432 kindly review :)

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**